### PR TITLE
Token Manager: small updates

### PR DESCRIPTION
**Pull request by sohkai**
Builds on #480.

A number of small things, changing some linting behaviour and mostly cosmetic code changes to align the Token Manager with the refactored Voting app in #483.

Easiest to review commit-by-commit.

### CODE REVIEW

File: apps/token-manager/app/src/components/SideBar.js [hunk 1]
```diff
@@ -16,12 +16,11 @@ const DISTRIBUTION_COLORS = [
 ]
 
 const displayedStakes = (accounts, total) => {
-  const positiveAccounts = accounts.filter(({ balance }) => balance > 0)
```
Comment (sohkai) [inline apps/token-manager/app/src/components/SideBar.js]: I think this should've actually been `balance.gtn(0)` anyway, but this is actually already [handled in the script](https://github.com/aragon/aragon-apps/blob/master/apps/token-manager/app/src/script.js#L102).

File: apps/token-manager/app/.eslintrc [hunk 2]
```diff
@@ -22,6 +21,7 @@
   },
   "plugins": ["prettier", "react", "jest"],
   "rules": {
+    "valid-jsdoc": "error",
```
Comment (bpierre) [inline apps/token-manager/app/.eslintrc]: Nice 👀

File: apps/token-manager/app/src/components/TokenBadge.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ class TokenBadge extends React.PureComponent {
   render() {
     const { address, name, symbol } = this.props
     return (
-      <Main title={name === symbol ? name : `${name} (${address})`}>
+      <Main title={`${name} (${address})`}>
```
Comment (bpierre) [inline apps/token-manager/app/src/components/TokenBadge.js]: 👍

File: apps/token-manager/app/src/utils.js [hunk 4]
```diff
@@ -113,9 +119,9 @@ export function stakesPercentages(
 
   // Remove the 0% items and group them in a “Rest” item.
   return hasRest
-    ? // A “Rest” item already exist, we can remove the 0% items.
+    ? // A “Rest” item already exists, we can remove the 0% items.
```
Comment (bpierre) [inline apps/token-manager/app/src/utils.js]: 🙈

File: apps/token-manager/app/src/utils.js [hunk 5]
```diff
@@ -113,9 +119,9 @@ export function stakesPercentages(
...
       adjustedStakes.slice(0, firstZeroIndex)
-    : // A “Rest” item need to be added and can not be zero,
+    : // A “Rest” item needs to be added and can not be zero,
```
Comment (bpierre) [inline apps/token-manager/app/src/utils.js]: 🙈 🙈

**Pull request merged**